### PR TITLE
Rares/1083 fix integrations

### PR DESCRIPTION
**Pull request by teodosii**
# What this PR does

- Fix for #1083 
- Fix for #1257 

- [x] `CHANGELOG.md` updated

### CODE REVIEW

File: grafana-plugin/src/pages/integrations/Integrations.tsx [hunk 1]
```diff
@@ -122,6 +128,7 @@ class Integrations extends React.Component<IntegrationsProps, IntegrationsState>
   }
 
   componentWillUnmount() {
+    this.isMounted = false;
     clearInterval(this.alertReceiveChannelTimerId);
```
Comment (maskin25) [inline grafana-plugin/src/pages/integrations/Integrations.tsx:132]: hmm, timer is cleared here, didn't get why should we check `this.isMounted`...

**Review by joeyorlando — CHANGES_REQUESTED** [review]
Are you also able to still reproduce the issue?
https://www.loom.com/share/1e81d6b1e2aa4b8ab7be24d3b7d38c59

**Comment by teodosii** [comment]
Thanks @joeyorlando - will further investigate - seems there's more around that

File: grafana-plugin/src/pages/integrations/Integrations.tsx (hunk 1 above)
Comment (teodosii) [reply grafana-plugin/src/pages/integrations/Integrations.tsx:132]: That has nothing to do with timer, it's about when you move from integrations to a separate page. Once it loads the escalation chains it calls that method but you're on a different page already.

**Review by joeyorlando — APPROVED** [review]
good stuff! aside from the `markdownlint` failing, LGTM 🚢

**Comment by joeyorlando** [comment]
@maskin25 are you okay with merging this in today? Will unblock #900

**Pull request merged**